ansible/group_vars/all.yml.example

```diff
@@ -36,8 +36,22 @@ notify_email_replyto: noreply@example.com
 # --------------------------------
 # OpenShift client command options
 # --------------------------------
+# Path to openshift client
+oc_path: "/usr/bin/oc"
```

Comment:
Well... this default can break some setups (like mine... I have ~/bin/oc symlinked to some version for easy switching). Can we make the default detect the current `oc` path?